src/RootApp.tsx

```diff
@@ -1,37 +1,53 @@
 import loadable from '@loadable/component';
+import { ThemeProvider } from '@mui/material/styles';
 import { History } from '@remix-run/router';
 import React from 'react';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
 
-import StableApp from './apps/stable/App';
-import { HistoryRouter } from './components/router/HistoryRouter';
-import { ApiProvider } from './hooks/useApi';
-import { WebConfigProvider } from './hooks/useWebConfig';
+import StableApp from 'apps/stable/App';
```

Comment:
Can this be load by ladable?

Reply:
It doesn't make too big of a difference since we aren't loading any additional large libraries for the stable layout (like mui in the experimental layout), but every little bit probably helps! :sweat_smile: